frontends/mit-learn/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -296,15 +307,15 @@ const SlickCarousel = () => {
 const TestimonialsSection: React.FC = () => {
   return (
     <Section>
-      <Container id="hamster-noises">
+      <HeaderContainer id="hamster-noises">
         <Typography component="h2" variant="h2">
           From Our Community
         </Typography>
-        <Typography variant="h3">
+        <Typography variant="body1">
           Millions of learners are reaching their goals with MIT's non-degree
           learning resources. Here's what they're saying.
         </Typography>
```

Comment:
Looking at figma, I guess this should have textAlign: center.

Could be `styled(...)`, or Typography has a prop for that.

Reply:
The `HeaderContainer` styled component handles this with `alignItems: "center"`, do we really need a duplicate centering rule?

Reply:
It's not duplicate, they do different things. 
- `align-items: center` aligns the purple box within the blue box
- `text-align: center` justifies the text at center instead of left

![Screenshot 2024-09-17 at 4 18 56 PM](https://github.com/user-attachments/assets/0cc41c39-2812-446a-aba2-0cacbc0a5b1d)